app/controllers/authorities_controller.rb

```diff
@@ -8,7 +8,11 @@ def query
       if params[:term] == "location"
         GeoNamesResource.find_location(s)
       else
-        LocalAuthority.entries_by_term(params[:model], params[:term], s) rescue []
+        begin
```

Comment:
Good question!

Rubocop hates one-liners with rescue [B]: https://github.com/bbatsov/ruby-style-guide#no-rescue-modifiers

It wanted the one-liner in [A] because of this: https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier